lib/diagnostic.js

```diff
@@ -39,8 +44,13 @@ const any = function (/* array, callback */) {
 // the callback returns a truthy value
 // you may use any feature of JavaScript
 // other than `Array.prototype.filter`
-const select = function (/* array, callback */) {
-
+const select = function (array, callback) {
+  let newArray = []
+  for (var i = 0; i < array.length; i++) {
+   if (any(array[i]) {
```

Comment:
This line will break the script.  There needs to be an additional closing parenthesis.